tom_queue.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.require_paths  = ['lib']
   spec.summary        = 'AMQP hook for ActiveRecord backend for DelayedJob'
   spec.test_files     = Dir.glob("spec/**/*")
-  spec.version        = '0.0.2'
-
+  spec.version        = TomQueue::VERSION
+  spec.executables    = ['deferred_work_manager']
```

Comment:
Our [style guide](https://github.com/fac/ruby-style-guide/) states:

> Prefer %w to the literal array syntax when you need an array of strings.

Can we adhere to it please?